examples/algorithms/rcm.py

```diff
@@ -14,7 +14,10 @@
 # BSD License
 import networkx as nx
 from networkx.utils import reverse_cuthill_mckee_ordering
-import numpy as np
+try:
```

Comment:
For the examples, I would just import things and let them fail.   I don't feel strongly about this, but it seems like the examples should be what people normally would write in a script and I don't think the should normally try to import.

Reply:
The modules in ``networkx`` (as opposed to ``examples``) should have numpy imports in a try statement, though.

Reply:
Done.